### ISSUE TITLE
Retarded. I don't even think these kinda activists deserve to be in technology

### ISSUE THREAD

**Issue by ghost** [CoralineAda/postmeritocracy#30]
This is trash. I and others can give 10^inf reasons.

* A person is actually defined by his work / talent. If there were no talented engineers and scientists, you would not have anything to spread trash conspiracy theories.

* Open source rocks **only because** nerds & scientific people did it, not marketing corporates.

* We don't even need mission critical things to be beginner friendly. We need quality, not participation. I am a newbie but this is what one should accept.

* We don't think everyone should participate. That is never the priority over productivity and code quality. Newbies have other channels to gain expertise and then enter.

Nerds have done things very well. Because science and technology are things we had in mind -- not utter popularity by saying trash like this, we did not thrive for power, we thrived for excellence.

If you are from social science trash, don't try to interfere here.

For a project like Linux kernel, the main pioneer being odd sometimes is **perfectly acceptable**. Linux is mission critical and code quality is thing, not LGBT participation.

TL;DR -- people who **suck at programming** do these kinda things to gain popularity.

**Comment by aphextwins** [comment]
Definition of meritocracy 
1 : a system in which the talented are chosen and moved ahead on the basis of their achievement
2 : leadership selected on the basis of intellectual criteria.

**Comment by Mooninaut** [comment]
If people were actually defined by their work and talent, there would not be systematic discrimination against women and PoC in STEM, and there definitely is.

"Nerds" have been systematically excluding women an PoC from their communities for decades (even though our communities were founded by women and PoC).

There is a difference between being "odd" and being hostile.

You are being hostile.

**Comment by sawyermade** [comment]
> If people were actually defined by their work and talent, there would not be systematic discrimination against women and PoC in STEM, and there definitely is.
> 
> "Nerds" have been systematically excluding women an PoC from their communities for decades (even though our communities were founded by women and PoC).
> 
> There is a difference between being "odd" and being hostile.
> 
> You are being hostile.

I have to disagree. There is no systematic discrimination in STEM against women and coloreds. That  is not only false but extremely dangerous to spread. There may be some company cultures that arent very inclusive but that is far, far, far from systemic. Do you even know the definition of systemic? In order to be systemic, it would have to be on a government/university/education level and that is not the case. STEM majors at the university level are constantly trying to recruit underrepresented demographics all the time and there is nothing stopping women or coloreds from joining STEM. In fact, it is quite the opposite. Also, wtf are you smoking, us nerds love having women around. It is a nice change of pace getting to work with women in classes and projects instead of your stereotypical "male nerd". No one is excluding women (though I do know one or two guys who prefer not to work with women if they dont have to) and especially not coloreds (Ive never met a single person who refused to work with someone based on race). You just cant make such divisive and sweeping generalizations without expecting to get checked for it. Just bc you say so or "feel" that its true, does not make it true. Just because there is underrepresentation doesnt automatically mean institutional discrimination lol. Ever heard of the Gender-equality paradox (https://en.wikipedia.org/wiki/Gender-equality_paradox) from Scandinavian countries? It basically shows that the more egalitarian a society is, the more it tends to be segregated by gender. It also proves that even though more women tested better for ability in STEM than men, they just end up doing what they are passionate about because they live in a stable and equal society, much like the United States. The same ppl that are alwasy complaining about underrepresentation in STEM are the same people who decided to study something else instead of actually going into STEM. I also never hear them complaining about how there are too many women in nursing and biomedical sciences that they need to actively recruit men. I also never hear them talking about getting young men and boys to even go to college, since women have been the majority of University students and post-graduate students, by quite a significant margin. Where is all the complaining for that? Is there a systemic bias against young men and boys too? Do you see how silly you sound?

**Comment by xkvcr** [comment]
> You just cant make such divisive and sweeping generalizations without expecting to get checked for it. Just bc you say so or "feel" that its true, does not make it true. Just because there is underrepresentation doesnt automatically mean institutional discrimination lol. Ever heard of the Gender-equality paradox (https://en.wikipedia.org/wiki/Gender-equality_paradox) from Scandinavian countries?

Interesting that you brought this up, because a) the study merely hinted at correlations, and b) you've made some sweeping generalizations too.
